### ISSUE TITLE
Pre-training model

### ISSUE THREAD

**Issue by 953585130** [2012013382/C3D-Tensorflow-slim#4]
Hi，
I probably looked at your train.py and c3d.py files. I want to know why your accuracy is so high without a pre-trained model. I'm confused. If I change to the HMDB51 dataset (only 51 classes), how should fc and softmax be modified?

Looking forward to your reply!

**Comment by 2012013382** [comment]
Sorry for the late reply！ In UCF-101, every action(the same protagonist) has several videos and they are very similar, I random chose 1/4 videos as the test set so that there are some videos which are similar as those in the train set. In the standard UCF-101 split, it avoids this setting.  I will modify it one day, and thank you for your question.
Best regard!
